lib/helper/WebDriverIO.js

```diff
@@ -704,6 +706,29 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Grab CSS property for given locator
+   *
+   * ```js
+   * I.grabCssPropertyFrom('h3', 'font-weight');
+   * ```
+   */
+  grabCssPropertyFrom(locator, cssProperty) {
```

Comment:
We leave it as it is now implemented. If there is several h3 on page it will returns array with h3 `font-weight` values